### PR TITLE
Fixes teardown issue with test_multiple_pvc_creation_after_deletion_performance

### PR DESCRIPTION
**Pull request by keesturam**

### CODE REVIEW

File: tests/e2e/test_pvc_creation_performance.py [hunk 1]
```diff
@@ -167,6 +165,15 @@ def test_multiple_pvc_creation_after_deletion_performance(
             number_of_pvc=number_of_pvcs,
             size=self.pvc_size,
         )
+        for pvc_obj in pvc_objs:
+            pvc_obj.reload()
+            teardown_factory(pvc_obj)
+        with ThreadPoolExecutor() as executor:
+            for pvc_obj in pvc_objs:
+                executor.submit(
```
Comment (jilju) [inline tests/e2e/test_pvc_creation_performance.py]: helpers.wait_for_resource_state will be started in a separate thread. The main thread will continue without actually waiting for the status.

**Comment by ebenahar** [comment]
@keesturam Would like to close old PRs, is this one still relevant?

**Comment by keesturam** [comment]
> @keesturam Would like to close old PRs, is this one still relevant?

This test was blocked due to performance improvements and was a low priority. But with 4.6, performance improvements are introduced so we'll have this open to be fixed.

**Comment by ramkiperiy** [comment]
@keesturam Is this PR still valid?, I guess it needs some re-work to resolve conflicts and rebase required.

**Comment by petr-balogh** [comment]
Closing due to inactivity! Feel free to re-open or send the new PR.

**Pull request closed without merging**